gtk/package-gtk-bin.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import shutil
+import subprocess
+import sys
+import tempfile
+from typing import List
+import types
+
+def parse_args(argv: List[str]) -> types.SimpleNamespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--source', help='GTK install prefix to copy',
+                        type=Path, default=Path.home() / "gtk" / "inst")
+    parser.add_argument('-o', '--output', help='Write tar file to this path',
+                        type=Path, required=True)
+    parser.add_argument('--zstd', action="store_true",
+                        help='Use zstd compression')
```

Comment:
Note that using `zstd` for compression will require a slightly different decompressing command in the pipeline (in `build_mac.yml`).
I guess we should use zstd compression for the pipeline, since the archive takes less space and the decompression is way faster.